websites/R/Reaper Scans/presence.ts

```diff
@@ -3,87 +3,104 @@ const presence = new Presence({
+			if (searchInput === document.activeElement) {
+				presenceData.details = "Searching";
+				presenceData.state = searchInput.value;
+				presenceData.smallImageKey = Assets.Search;
+			} else presenceData.details = "Viewing Home Page";
+			break;
+		}
+		case "comics":
+		case "novels": {
+			const captitalized = `${pathSplit[0][0].toUpperCase()}${pathSplit[0].slice(
+				1
+			)}`;
+			if (pathSplit[1]) {
+				if (pathSplit[2] === "chapters") {
+					let progress =
+						(document.documentElement.scrollTop /
+							(document.querySelector(
+								pathSplit[0] === "comics"
+									? "main > div:not([id*='Ads'])"
+									: "article"
+							).scrollHeight -
+								window.innerHeight)) *
+						100;
+					progress = Math.ceil(progress) > 100 ? 100 : Math.ceil(progress);
 
-	if (
-		document.querySelector(".search-main-menu").classList.contains("active")
-	) {
-		presenceData.details = "Searching:";
-		presenceData.state = (
-			document.querySelector(".manga-search-field") as HTMLInputElement
-		).textContent;
-		presenceData.smallImageKey = "search";
-	} else if (/^\/$/.test(pathname)) presenceData.details = "Viewing Home Page";
-	else if (/^\/home1\/?$/.test(pathname)) {
-		// Counting comics
-		comics = document.querySelectorAll(".page-listing-item .row .col-4").length;
-		presenceData.details = "Viewing Comic List";
-		presenceData.state = `📋 ${comics.toString()} comics found`;
-	} else if (/^\/all-series\/novels+\/?$/.test(pathname)) {
-		// Counting novels
-		comics = document.querySelectorAll(".page-listing-item .row .col-6").length;
-		presenceData.details = "Viewing Novel List";
-		presenceData.state = `📋 ${comics.toString()} novels found`;
-	} else if (/^\/all-series\/comics\/manhwas\/?$/.test(pathname)) {
-		// Counting manhwa
-		comics = document.querySelectorAll(".page-listing-item .row .col-6").length;
-		presenceData.details = "Viewing Manhwa List";
-		presenceData.state = `📋 ${comics.toString()} manhwa found`;
-	} else if (/^\/all-series\/comics\/manhuas\/?$/.test(pathname)) {
-		// Counting manhua
-		comics = document.querySelectorAll(".page-listing-item .row .col-4").length;
-		presenceData.details = "Viewing Manhua List";
-		presenceData.state = `📋 ${comics.toString()} manhua found`;
-	} else if (/^\/series\/[0-9a-z-]+\/?$/i.test(pathname)) {
-		presenceData.details = "Viewing Comic";
-		presenceData.state = document.querySelector(".post-title h1").textContent;
-		presenceData.smallImageKey = "view";
-		presenceData.buttons = [
-			{
-				label: "Visit Comic Page",
-				url: origin + pathname,
-			},
-		];
-	} else if (
-		/^\/series\/[0-9a-z-]+\/+(chapter|ch)-[0-9]+\/?$/i.test(pathname)
-	) {
-		let progress =
-			(document.documentElement.scrollTop /
-				(document.querySelector(".read-container").scrollHeight -
-					window.innerHeight)) *
-			100;
-		progress = Math.ceil(progress) > 100 ? 100 : Math.ceil(progress);
+					presenceData.details = `Reading ${document
```

Comment:
remove `Reading`, show only the title

Reply:
I disagree with removing the action from the title. Many other similar presences have the same format.

Reply:
Agreed with @theusaf